src/MassTransit/Configuration/PipeConfigurators/DiagnosticsActivityConsumePipeSpecification.cs

```diff
@@ -16,24 +16,23 @@ namespace MassTransit.PipeConfigurators
 {
     using System.Collections.Generic;
     using System.Diagnostics;
-    using System.Linq;
     using GreenPipes;
-    using Pipeline.Filters;
     using Pipeline.Filters.DiagnosticActivity;
 
+
     public class DiagnosticsActivityConsumePipeSpecification<T> :
         IPipeSpecification<ConsumeContext<T>>
         where T : class
     {
-        readonly DiagnosticSource _diagnosticSource;
-        readonly string _activityIdKey;
         readonly string _activityCorrelationContextKey;
+        readonly string _activityIdKey;
+        readonly DiagnosticSource _diagnosticSource;
 
         public DiagnosticsActivityConsumePipeSpecification(DiagnosticSource diagnosticSource, string activityIdKey, string activityCorrelationContextKey)
         {
-            _diagnosticSource = diagnosticSource;
-            this._activityIdKey = activityIdKey;
-            this._activityCorrelationContextKey = activityCorrelationContextKey;
+            _diagnosticSource              = diagnosticSource;
```

Comment:
where did this annoying reformat come from? not my settings.

Reply:
I think it was a blip. It shouldn't have come from mine either as i generally have it off